src/screens/createPost/index.js

```diff
@@ -0,0 +1,202 @@
+          <img src={url} alt="..." width="200" height="100" border="3" />
+        ))}
+      </div>
+      <Button
+        variant="contained"
+        color="primary"
+        component="span"
+        startIcon={<CloudUploadIcon />}
+        onClick={handleOpen}
+      >
+        Upload Images
+      </Button>
+      <DropzoneDialog
+        open={open}
+        onSave={handleSave}
+        acceptedFiles={["image/jpeg", "image/png", "image/bmp"]}
+        showPreviews={true}
+        maxFileSize={5000000}
+        onClose={handleClose}
+      />
+    </div>
+  );
+}
+
+export default function Maxmised(props) {
+  const classes = useStyles();
+  var imgarr = [];
+  const [text1, setText1] = useState(false);
+  const [text2, setText2] = useState(false);
+
+  return (
+    <Card className={classes.card}>
+      <CardHeader
+        classes={{ title: classes.title, subheader: classes.subheader }}
+        avatar={
+          <Avatar className={classes.avatar}>
+            <CreateIcon color="primary" fontSize="medium" />
+          </Avatar>
+        }
+        title="CREATE POST"
+        subheader={props.stream}
+        style={{
+          textAlign: "left",
+          backgroundColor: "#3f51b5",
+        }}
+      />
+
+      <CardContent>
+        <Typography variant="body2" color="black" component="p">
+          <form className={classes.root} noValidate autoComplete="off">
+            <div>
+              <TextField
+                required
+                id="post-title"
+                label="Post Title"
+                placeholder="Your Title"
+                variant="outlined"
+                fullWidth
+                error={text1}
+              />
+            </div>
+            <div>
+              <TextField
+                required
+                id="post-content"
+                label="Post Content"
+                placeholder="Your Content"
+                multiline
+                rows="12"
+                variant="outlined"
+                error={text2}
+              />
+            </div>
+            <div className={classes.root}>
+              <DropzoneDialogExample filearray={imgarr} />
+            </div>
+            <div>
+              <Button
+                variant="contained"
+                color="primary"
+                className={classes.button}
+                endIcon={<SendIcon />}
+                onClick={() => {
+                  if (document.getElementById("post-title").value === "")
+                    setText1(true);
+                  else setText1(false);
+                  if (document.getElementById("post-content").value === "")
+                    setText2(true);
+                  else setText2(false);
+                  if (
+                    !(
+                      document.getElementById("post-title").value === "" ||
+                      document.getElementById("post-content").value === ""
+                    )
+                  ) {
```

Comment:
Instead of using the `document` API, you can use a React ref - see [`React.useRef`](https://reactjs.org/docs/hooks-reference.html#useref). Essentially, refs are for maintaining a direct reference to DOM elements.